### PR TITLE
Changelog for v0.10.0

### PR DESCRIPTION
**Pull request by stuartnelson3**
@grobie

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+## 0.10.0 / 2017-11-09
+* [ENHANCEMENT] Improve amtool check-config use and description text (#1016)
```
Comment (grobie) [inline CHANGELOG.md]: Empty after the heading.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Add metrics about current silences and alerts (#998)
+* [ENHANCEMENT] Sorted silences based on current status (#1015)
+* [ENHANCEMENT] add metric of alertmanager position in mesh (#1024)
+* [ENHANCEMENT] Initialise notifications_total and notifications_failed_total (#1011)
+* [ENHANCEMENT] Allow selectable matchers on silence view (#1030)
+* [ENHANCEMENT] Rename Promtheus to Alertmanager in CONTRIBUTING.md (#1033)
```
Comment (grobie) [inline CHANGELOG.md]: Remove, not interesting for users.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Allow template in victorops message_type field (#1038)
+* [ENHANCEMENT] Optionally hide inhibited alerts in API response (#1039)
+* [ENHANCEMENT] Improve config test coverage (#1046)
```
Comment (grobie) [inline CHANGELOG.md]: Remove.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Convert Alertmanager to use non-global go-kit loggers (#1052)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Add tests to notifiers config (#1050)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Toggle silenced and inhibited alerts in UI (#1049)
+* [ENHANCEMENT] Fix pushover limits (title, message, url) (#1055)
+* [ENHANCEMENT] Add limit to OpsGenie message (#1045)
+* [ENHANCEMENT] Add logging to statemessage truncation (#1056)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Upgrade OpsGenie notifier to v2 API. (#1061)
+* [ENHANCEMENT] Fix documentation (#1065)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Allow enabling mutex and block profiles (#1073)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Allow template in victorops routing_key field (#1083)
+* [ENHANCEMENT] Add support for PagerDuty API v2 (#1054)
+* [ENHANCEMENT] Prevent alerts from inhibiting themselves (#1017)
+* [BUGFIX] dispatch: fix race condition in dispatch test (#1025)
```
Comment (grobie) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md (hunk 2 above)
Comment (stuartnelson3) [reply CHANGELOG.md]: 👍 i started dropping things halfway through, should have cleaned up the previous ones

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,35 @@
...
+* [ENHANCEMENT] Toggle silenced and inhibited alerts in UI (#1049)
+* [ENHANCEMENT] Fix pushover limits (title, message, url) (#1055)
+* [ENHANCEMENT] Add limit to OpsGenie message (#1045)
+* [ENHANCEMENT] Add logging to statemessage truncation (#1056)
+* [ENHANCEMENT] Upgrade OpsGenie notifier to v2 API. (#1061)
+* [ENHANCEMENT] Fix documentation (#1065)
+* [ENHANCEMENT] Allow enabling mutex and block profiles (#1073)
+* [ENHANCEMENT] Allow template in victorops routing_key field (#1083)
+* [ENHANCEMENT] Add support for PagerDuty API v2 (#1054)
+* [ENHANCEMENT] Prevent alerts from inhibiting themselves (#1017)
```
Comment (brian-brazil) [inline CHANGELOG.md]: This is a CHANGE, and more accurately it prevents inhibiting alerts in the source of the inhibition.

**Comment by stuartnelson3** [comment]
@grobie addressed

File: CHANGELOG.md (hunk 10 above)
Comment (grobie) [reply CHANGELOG.md]: @stuartnelson3

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,28 @@
...
+* [ENHANCEMENT] Allow selectable matchers on silence view (#1030)
+* [ENHANCEMENT] Allow template in victorops message_type field (#1038)
+* [ENHANCEMENT] Optionally hide inhibited alerts in API response (#1039)
+* [ENHANCEMENT] Toggle silenced and inhibited alerts in UI (#1049)
+* [ENHANCEMENT] Fix pushover limits (title, message, url) (#1055)
+* [ENHANCEMENT] Add limit to OpsGenie message (#1045)
+* [ENHANCEMENT] Upgrade OpsGenie notifier to v2 API. (#1061)
+* [ENHANCEMENT] Allow template in victorops routing_key field (#1083)
+* [ENHANCEMENT] Add support for PagerDuty API v2 (#1054)
+* [CHANGE] Prevent inhibiting alerts in the source of the inhibition (#1017)
```
Comment (grobie) [inline CHANGELOG.md]: CHANGEs go to the top. See https://github.com/prometheus/prometheus/wiki/HOWTO-cut-a-new-release#prepare-your-release

Comment (stuartnelson3) [reply CHANGELOG.md]: release fail, thanks for being diligent in checking this

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,28 @@
+## 0.10.0 / 2017-11-09
+
+* [CHANGE] Prevent inhibiting alerts in the source of the inhibition (#1017)
+* [ENHANCEMENT] Improve amtool check-config use and description text (#1016)
+* [ENHANCEMENT] Add metrics about current silences and alerts (#998)
+* [ENHANCEMENT] Sorted silences based on current status (#1015)
+* [ENHANCEMENT] add metric of alertmanager position in mesh (#1024)
```
Comment (brancz) [inline CHANGELOG.md]: nit: add -> Add (for consistency with the other entries)

**Pull request merged**